README.md

```diff
@@ -111,3 +111,29 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Run:
+```Shell
+tools/setup_env.sh
```

Comment:
I was concerned about the issues with re-running setup_env.sh anyway. I'm inclined to just provide installation instructions in the README, and not run the commands in setup_env?